### ISSUE TITLE
Want some way to mount kbfs as nobrowse

### ISSUE THREAD

**Issue by kballard** [keybase/client#1901]
Having the `/keybase` volume show up on my desktop (OS X) is irritating. I'd prefer to have it mounted with the `nobrowse` option.

I tried running `mount -u -o nobrowse /keybase`, but all that did was show me the output

```
KBFUSE mount version 3.1.0

This program is not meant to be called directly. The KBFUSE library calls it.

Available mount options:
...
```

**Comment by strib** [comment]
Hi @kballard, thanks for the suggestion.  I will put this on our internal tracker.  I'm guessing it will be easy to provide an option for this; the harder part will be how, if we decide the keep the default behavior as is, to give you the option.  We don't really have a good story for config files or passing in command line params on OSX right now (though we probably should).

**Comment by kballard** [comment]
Since I'm running Keybase.app which is sitting in my menubar, you could add a settings pane to it maybe?

For CLI usage, there could be a `keybase` subcommand that lets you twiddle nobrowse on/off, which could write to some standard location (either to `NSUserDefaults` to make it easy to share with Keybase.app, or maybe to `~/.config/keybase/`).

**Comment by strib** [comment]
Yep, agreed.  There are lots of good ways to do this, we just have to settle on the right way for Keybase, for this issue and plenty of other config issues that will come up, I'm sure.
